frontend/lib/api/form/pollingstation/PollingStationFormController.tsx

```diff
@@ -164,6 +169,15 @@ export function PollingStationFormController({
     }
   }, [doRequest, values]);
 
+  const deleteDataEntry = async () => {
+    const path = `/api/polling_stations/${pollingStationId}/data_entries/${entryNumber}`;
```

Comment:
There optionally is a typed template literal for paths from the type generation

Reply:
const path:UGLY_SNAKE_UPPERCASE = `/api/${param}`;
it at least gives typo checking. but a helper function would be better I agree.

Reply:
Changed in cf58815fdf7b4c16061268956cbb438006ad1519.